library/src/main/kotlin/com/chuckerteam/chucker/internal/support/ResponseExt.kt

```diff
@@ -0,0 +1,17 @@
+package com.chuckerteam.chucker.internal.support
+
+import okio.BufferedSource
+import java.util.regex.Matcher
+import java.util.regex.Pattern
+
+private const val IP_REGEX = "(?:\\d{1,3}\\.){3}\\d{1,3}"
+
+public fun BufferedSource.getHostIp(): String? {
+    val body = readUtf8()
```

Comment:
This reads the whole body of the source to the memory and assumes that it can be read as a UTF-8 stream.